Problem2.scala

```diff
@@ -0,0 +1,53 @@
+import org.apache.spark.SparkConf
+import org.apache.spark.sql.SparkSession
+import org.apache.spark.sql.functions._
+
+
+
+object Problem2 extends App{
```

Comment:
Object name as movie rating

Reply:
object name made to topMovies instead of Problem2